docs/tutorials/io.rst

```diff
@@ -1,31 +1,38 @@
-Input/Output
-------------
+Input/Output Pins
+-----------------
 
 There are strips of metal along the bottom edge of the BBC micro:bit that make
 it look as if the device has teeth. These are the input/output pins (or I/O pins
 for short).
 
 .. image:: blue-microbit.png
+    :width: 300px
+    :align: center
+    :height: 270px
+    :alt: micro:bit with pins labelled
 
 Some of the pins are bigger than others so it's possible to attach crocodile
 clips to them. These are the ones labelled 0, 1, 2, 3V and GND (computers
 always start counting from zero). If you attach an edge connector board to the
 device it's possible to plug in wires connected to the other (smaller) pins.
 
-Each pin on the BBC micro:bit is represented by an *object* called ``pinN``
-where ``N`` is the pin number. So, for example, to do things with the pin
-labelled with a 0 (zero), use the object called ``pin0``.
+On the latest micro:bit **V2** the micro:bit logo can also be used as a touch
+input.
 
-Simple!
+In MicroPython, each pin on the BBC micro:bit is represented by an *object*
+called ``pinN``, where ``N`` is the number pf the pin. 
+
+For example, to use the pin labelled 0 (zero), you can use the object called
+``pin0`` in your script. The logo pin **V2** uses ``pin_logo``.
 
 These objects have various *methods* associated with them depending upon what
-the specific pin is capable of.
+the specific `pin is capable of <Pin Functions>`_ eg. read, write or touch.
```

Comment:
I've removed this link. I'm not entirely sure how referencing works I was hoping it would link here https://github.com/bbcmicrobit/micropython/pull/709/files#diff-f6860010c228528bb564ff9d1253ec9692154196c9369a9c7e52c24b4e1d3249R23 to a Pin Functions target, but it doesn't so I'm happy to leave it as it was. I think if we were to review the tutorials in more depth, we could be interlinking more with the API for context.